client/src/plugins/update-checks/UpdateChecks.js

```diff
@@ -335,16 +366,49 @@ export default class UpdateChecks extends PureComponent {
     this.onClose();
   };
 
+  openVersionInfoPage = () => {
+    this.setState({
+      showModal: true,
+    });
+  };
+
   render() {
     const {
       showModal,
       latestVersionInfo,
       currentVersion,
-      updateChecksEnabled
+      updateChecksEnabled,
+      open,
+      updateAvailable
     } = this.state;
 
+    const {
+      toggle,
+      _buttonRef: buttonRef,
+      close,
+      openVersionInfoPage,
+    } = this;
     return (
       <React.Fragment>
+        {updateAvailable && <Fill slot="status-bar__app" group="9_update_checks">
+          <button
+            style={ {
+              backgroundColor: 'rgba(0, 119, 204, 1)',
+              color: 'rgb(254, 254, 254)'
+            } }
+            className="btn btn-primary"
+            title="Toggle update info"
+            onClick={ toggle }
+            ref={ buttonRef }
+          >{ 'Update' } </button>
+        </Fill>}
+        {open && <UpdateAvailableOverlay
```

Comment:
Consider renaming `open` to clarify what is open as this component handles multiple things.